pkgs/tools/virtualization/multipass/default.nix

```diff
@@ -0,0 +1,93 @@
+{ cmake
+, dnsmasq
+, fetchFromGitHub
+, git
+, iproute2
+, iptables
+, lib
+, libapparmor
+, libvirt
+, libxml2
+, nixosTests
+, openssl
+, OVMF
+, pkg-config
+, qemu
+, qemu-utils
+, qtbase
+, qtx11extras
+, slang
+, stdenv
+, wrapQtAppsHook
+, xterm
+}:
+
+let
+  pname = "multipass";
+  version = "1.11.0";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "canonical";
+    repo = "multipass";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-2d8piIIecoSI3BfOgAVlXl5P2UYDaNlxUgHXWbnSdkg=";
+    fetchSubmodules = true;
+  };
+
+  preConfigure = ''
+    substituteInPlace ./CMakeLists.txt \
+      --replace "determine_version(MULTIPASS_VERSION)" "" \
+      --replace 'set(MULTIPASS_VERSION ''${MULTIPASS_VERSION})' 'set(MULTIPASS_VERSION "v${version}")'
+
+    substituteInPlace ./src/platform/backends/qemu/linux/qemu_platform_detail_linux.cpp \
+      --replace "OVMF.fd" "${OVMF.fd}/FV/OVMF.fd"
+  '';
+
+  buildInputs = [
+    qtbase
+    libapparmor
+    qtx11extras
+    libvirt
+    libxml2
+    openssl
+  ];
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+    cmake
+    git
+    pkg-config
+    slang
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
I'm surprised this needs to be passed at all, do you think it didn't work on the first try without it?

Reply:
This will be removed in the next commit!